src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    operate(operation, num) {
```

Comment:
[CODE STYLE] - The `operate` method should use the result of the operation callback to update the `result` property. Use the assignment operator `=` along with the callback to ensure the `result` is updated correctly.